addons/website_event/views/event_templates.xml

```diff
@@ -546,10 +546,8 @@
                         </div>
                     </main>
                     <footer class="modal-footer">
-                        <div class="float-left">
-                            <button type="submit" class="btn btn-primary">Continue</button>
-                            <button type="button" class="btn btn-secondary js_goto_event" data-dismiss="modal">Cancel Registration</button>
-                        </div>
+                        <button type="button" class="btn btn-secondary js_goto_event" data-dismiss="modal">Cancel</button>
+                        <button type="submit" class="btn btn-primary">Continue Registration</button>
```

Comment:
Is this wanted you changed the labels ?

Reply:
Yeah it is just weird to have a bigger cancel button, it is like 3x the size of the important one, moving `Registration` into the Continue makes more sense imho

Reply:
Btw so you can see:
![event](https://user-images.githubusercontent.com/30048408/50688286-14c80d00-1025-11e9-9b4a-f3c302cc6b85.png)

Reply:
:+1: